courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb

```diff
@@ -41,6 +41,7 @@
    "metadata": {},
```

Comment:
<p>Line 1, "plots".</p><p>Line 2, "finds"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
> Line 1, "plots".
> 
> Line 2, "finds"
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1266/discussion/)_

Thanks Gwendolyn,
Updated this and all above suggested changes.